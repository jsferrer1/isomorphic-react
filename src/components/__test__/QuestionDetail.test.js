describe('The QuestionDetail component', () => {

  it('should not regress', () => {
  });
  
})
