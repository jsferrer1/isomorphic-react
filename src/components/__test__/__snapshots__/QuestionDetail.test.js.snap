// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Question Detail component The display element should not regress 1`] = `[Function]`;
